requirements.txt

```diff
@@ -5,6 +5,6 @@ pymc==2.3.4
 pyzmq>=13.1.0
 scipy>=0.12.0
 tornado>=3.0.2
-wsgiref>=0.1.2
+wsgiref>=0.1.2; python_version < '2.7'
```

Comment:
does this mean strictly less than 2.7, so _not_ including 2.7?

Reply:
Yeah, I cribbed it from a blog post about this issue elsewhere. Maybe `< 3.0` is a better idea?

Reply:
I think so yea

Reply:
@CamDavidsonPilon I think this is now sorted and OK to merge.